doc/links.inc

```diff
@@ -174,6 +174,11 @@
 .. _travis: https://travis-ci.org/
 .. _linterflake8: https://atom.io/packages/linter-flake8
 .. _Brain Imaging Data Structure: https://bids.neuroimaging.io/
+.. _NAS: https://en.wikipedia.org/wiki/Nasion
+.. _LPA: http://www.fieldtriptoolbox.org/faq/how_are_the_lpa_and_rpa_points_defined/
+.. _RPA: http://www.fieldtriptoolbox.org/faq/how_are_the_lpa_and_rpa_points_defined/
+.. _Polhemus: https://polhemus.com/scanning-digitizing/digitizing-products/
+.. _FieldTrip FAQ on coordinate systems: http://www.fieldtriptoolbox.org/faq/how_are_the_different_head_and_mri_coordinate_systems_defined/
```

Comment:
Not a big deal, but I've been putting one-off links at the bottom of the file where they are used, and only adding links here if they're referenced from multiple doc files. I think it's slightly easier to maintain, because future editors of the tutorial won't need to open a separate file to figure out where the link goes.  But I don't insist everyone should do it this way... use your judgment.

Reply:
I agree that this is a more tidy way to organize links. Thanks!